lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -2072,9 +2072,14 @@ diagnoseMatch(ModuleDecl *module, NormalProtocolConformance *conformance,
                    (unsigned)match.MissingRequirement->getKind());
     break;
 
-  case MatchKind::ThrowsConflict:
-    diags.diagnose(match.Witness, diag::protocol_witness_throws_conflict);
+  case MatchKind::ThrowsConflict: {
+    auto witness = match.Witness;
+    auto diag = diags.diagnose(witness, diag::protocol_witness_throws_conflict);
+    assert(isa<FuncDecl>(witness));
+    auto FD = cast<FuncDecl>(witness);
+    diag.fixItRemove(FD->getThrowsLoc());
```

Comment:
I'm not sure this is a good fix-it to offer. `throws` is unlikely to be accidentally added, and if there's anything relying on it in the body of the function we'd just get into an error loop.